### ISSUE TITLE
Homebrew GDAL

### ISSUE THREAD

**Issue by zakjan** [Esri/lerc#231]
Currently GDAL installed from Homebrew doesn't support LERC.

Is it possible to publish LERC to Homebrew, and link it with libtiff there?

See https://github.com/Homebrew/homebrew-core/issues/106450 for how ZSTD support was enabled.

**Comment by tmaurer3** [comment]
@zakjan  AFAIK both WebP and Lerc were added to libtiff in 2021. Gdal at this time already supported Lerc and certain combinations of Lerc with other, additional lossless compressors. You can find more on this here:  https://gdal.org/drivers/raster/gtiff.html .  So this should be a matter of setting the right options in Homebrew. In case of problems, the Gdal experts are your best bet.

**Comment by jshrake** [comment]
You can add LERC support to the homebrew installed gdal by modifying the `libtiff` formula and rebuilding it from source. 
  
1. Clone and build <https://github.com/Esri/lerc.git>

    ```bash
    cd /Users/justin/code
    git clone https://github.com/Esri/lerc.git
    cd lerc
    cmake -S. -Bbuild -DCMAKE_BUILD_TYPE=Release
    cmake --build build
    ```

2. Edit the homebrew `libtiff` formula with the following patch:

    ```diff
    diff --git a/Formula/libtiff.rb b/Formula/libtiff.rb
    index 4e20a84bb74..2ec59f64314 100644
    --- a/Formula/libtiff.rb
    +++ b/Formula/libtiff.rb
    @@ -33,7 +33,9 @@ class Libtiff < Formula
        args = %W[
        --prefix=#{prefix}
        --enable-zstd
    +   --enable-lerc
    +   --with-lerc-include-dir=/Users/justin/code/lerc/src/LercLib/include
    +   --with-lerc-lib-dir=/Users/justin/code/lerc/build
        --disable-lzma
        --disable-webp
        --with-jpeg-include-dir=#{Formula["jpeg-turbo"].opt_include}
    ```

    Replace the paths `with-lerc-include-dir` and `with-lerc-lib-dir` with the path you cloned above.
    Then, copy the patch and apply it:

    ```bash
    cd "$(brew --repository homebrew/core)"
    pbpaste | patch -p1
    brew reinstall --build-from-source libtiff
    git restore Formula/libtiff.rb
    ```

3. Your homebrew installed `gdal` should now have the LERC codec available:

    ```bash
    gdal_translate -of GTIFF -co COMPRESS=LERC -co MAX_Z_ERROR=1 input.tif output.tif
    ```

**Comment by jshrake** [comment]
I created a PR to add a formula for lerc to homebrew https://github.com/Homebrew/homebrew-core/pull/121231. Once this is merged, I can make a followup PR to enable lerc in the `libtiff` formula, which also enables lerc in the homebrew installed GDAL.

**Comment by jshrake** [comment]
PR to enable lerc in libtiff https://github.com/Homebrew/homebrew-core/pull/121300
